main.go

```diff
@@ -131,6 +137,11 @@ func StartService(c *cli.Context) {
 		log.Fatalf("Failed to reload the auth provider from db on start: %v", err)
 	}
 
+	err = server.UpgradeCase()
+	if err != nil {
+		log.Fatalf("Failed on upgrade case")
```

Comment:
you need to include the error in the log `log.Fatalf("Failed on upgrade case: %s", err)`